Source/DafnyLanguageServer.Test/Lookup/HoverTest.cs

```diff
@@ -597,7 +602,7 @@ class C {
   C.m(); // TODO
  //  ^[Unformatted comment] // Does not work yet.
   var c: C := new C();
-//                ^[The class C. Should be used like this:\n```dafny\nnew C();\n```]
+//                ^[This is the constructor\n```dafny\nconstructor C()\n```]
```

Comment:
Nice. Clicking on the symbol now redirect to the constructor instead of the class, I presume?

Reply:
I think it already did that